karrot/activities/activity_types.py

```diff
@@ -0,0 +1,28 @@
+from django.utils.translation import gettext as _
+
+default_activity_types = {
+    'Pickup': {
+        'name': _('Pickup'),
```

Comment:
Could/should/maybe store these names outside/seperate from the different types, as they are slightly independent from the defaults...